packages/openapi-generator/src/generator.ts

```diff
@@ -64,7 +78,12 @@ export async function generate(options: GeneratorOptions): Promise<void> {
     return generateService(
       inputFilePath,
       options,
-      vdmMapping,
+      getOrCreateServiceConfig(
```

Comment:
ah here it is. For me it would be cleaner to use a readOrGetDefault in line 70 and not not change the value behind the const `perServiceConfig`.

Reply:
Let's keep this one for discussion: https://github.com/SAP/cloud-sdk-js/pull/1249#discussion_r622987222